js/src/components/google-combo-account-card/account-card-description/account-details.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useGoogleAccount from '.~/hooks/useGoogleAccount';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
+
+/**
+ * Account details.
+ * @return {JSX.Element} JSX markup.
+ */
+const AccountDetails = () => {
+	const { google } = useGoogleAccount();
+	const { googleAdsAccount } = useGoogleAdsAccount();
+	const { googleMCAccount } = useGoogleMCAccount();
+
+	return (
+		<div className="gla-account-card__account_details">
+			<span>{ google.email }</span>
```

Comment:
@ankitrox Let's use the `p` tag. Thus we won't need the styles in `js/src/components/google-combo-account-card/connected-google-combo-account-card.scss` except to set the `margin: 0` to `p`